Blog posts: A journey with habitat on ECS parts 1 and 2

irvingpop opened the pull request:
TODO:

- [x] Figure out how to add a bio entry
- [x] Find an acceptance site (if one exists) so I can preview these posts

----

These are being ported over from my Medium posts:
- https://medium.com/@irvingpop/a-journey-with-habitat-on-amazon-ecs-part-1-39ae8af6542
- https://medium.com/@irvingpop/a-journey-with-habitat-on-amazon-ecs-part-2-1115c0463f83

thesentinels commented:
Thanks for the pull request! Here is what will happen next:
 1. Your PR will be reviewed by the maintainers
 2. If everything looks good, one of them will `approve` it, and your PR will be merged.

Thank you for contributing!

christophermaier commented:
@irvingpop You can add your bio information to [this file](https://github.com/habitat-sh/habitat/blob/master/www/data/author_bios.yml)

Also, once your posts are ready, one of the core team members can tell our bot to deploy this to our acceptance environment, where you can check it out (https://www.acceptance.habitat.sh/blog/). Once you're satisfied, we can get these merged!

irvingpop commented:
thanks @christophermaier !   Added a bio entry for myself, rebased and pushed.   Could somebody please deploy to acceptance?

christophermaier commented:
@irvingpop sure thing!

@thesentinels acceptance

thesentinels commented:
:metal: I am sending your branch against master to our acceptance environment.

thesentinels commented:
:neckbeard: Travis CI has [started the deployment of acceptance](https://travis-ci.org/habitat-sh/habitat/builds/275144481).

thesentinels commented:
:sparkling_heart: Travis CI [reports acceptance has been deployed](https://travis-ci.org/habitat-sh/habitat/builds/275144481).

I love new and shiny websites!

I just want you and the contributor to answer me one question:

![gif-keyboard-3280869874741411265](https://cloud.githubusercontent.com/assets/4304/17755674/1e577b82-6490-11e6-96b7-1663d283c824.gif)

irvingpop commented:
Thanks @christophermaier !  Looks good in acceptance.  I fixed one small typo and a link - I've removed the [WIP] flag from this, ready to ship when y'all are

christophermaier commented:
@irvingpop weeeeeeeeee!

@thesentinels approve

thesentinels commented:
:metal: I am testing your branch against master before merging it. We do this to ensure that the master branch is never failing tests.

thesentinels commented:
:neckbeard: Travis CI has [started testing this PR](https://travis-ci.org/habitat-sh/habitat/builds/275192644).

thesentinels commented:
:sparkling_heart: Travis CI [reports this PR passed](https://travis-ci.org/habitat-sh/habitat/builds/275192644).

It always makes me feel nice when humans approve of one anothers work. I'm merging this PR now.

I just want you and the contributor to answer me one question:

![gif-keyboard-3280869874741411265](https://cloud.githubusercontent.com/assets/4304/17755674/1e577b82-6490-11e6-96b7-1663d283c824.gif)
